index.html

```diff
@@ -203,10 +203,10 @@ <h2>CONTACT US</h2>
         a feature that you need built or a project
         that needs coding. I’d love to help with it.</p>
 
-      <form action="https://formspree.io/f/xayaljee" method="POST">
-        <input type="text" name="user_name" placeholder="Text input" maxlength="30" value="" required />
-        <input type="email" name="_replyto" placeholder="jeremy@gmail.com" required />
-        <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+      <form action="https://formspree.io/f/xayaljee" method="POST" id="contact-form">
+        <input type="text" id="username" name="user_name" placeholder="Text input" maxlength="30" value="" required />
+        <input type="email" id="useremail" name="_replyto" placeholder="jeremy@gmail.com" required />
+        <textarea name="message" id="usermsg" placeholder="Enter text here" maxlength="500" required></textarea>
```

Comment:
As you can see the data inside the inputs form is not the same as you can see in the Local Storage, you should add the functionality to update the Local Storage at the same time when I'm editing the inputs or textarea elements in the contact form.

- [ ] Your LocalStorage is not working properly, it would be nice if you can fix it.

![image](https://user-images.githubusercontent.com/60164239/132020877-9c4042b0-00c8-4351-8f6e-6c8969f723c5.png)